src/main/java/uk/gov/companieshouse/itemgroupworkflowapi/interceptor/UserAuthenticationInterceptor.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.itemgroupworkflowapi.interceptor;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.HandlerInterceptor;
+import uk.gov.companieshouse.itemgroupworkflowapi.logging.LoggingUtilsConfiguration;
+import uk.gov.companieshouse.itemgroupworkflowapi.util.EricHeaderHelper;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.springframework.http.HttpStatus.UNAUTHORIZED;
+import static uk.gov.companieshouse.itemgroupworkflowapi.logging.LoggingUtilsConfiguration.*;
+
+@Component
+public class UserAuthenticationInterceptor implements HandlerInterceptor {
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoggingUtilsConfiguration.APPLICATION_NAMESPACE);
```

Comment:
Perhaps a `LoggingUtils` bean can be injected as we already do in other places in this app?